on-the-web/index.html

```diff
@@ -14,30 +14,30 @@
                 <div class="social-media-feature-section-inner">                
                     <div class="social-media-feature-section-feature top-left">
-                        <a class="account-profile" href="http://www.twitter.com/openmicroscopy" target="_blank"><img class="account-profile-pic" src="https://pbs.twimg.com/profile_images/430266361785229313/MU4Sx0sn_400x400.png" alt="@openmicroscopy twitter profile pic"></a>
+                        <a class="account-profile" href="https://www.twitter.com/openmicroscopy" target="_blank"><img class="account-profile-pic" src="https://pbs.twimg.com/profile_images/430266361785229313/MU4Sx0sn_400x400.png" alt="@openmicroscopy twitter profile pic"></a>
                         <div>
-                            <p class="account-handle"><a href="http://www.twitter.com/openmicroscopy" target="_blank">@openmicroscopy</a></p>
+                            <p class="account-handle"><a href="https://www.twitter.com/openmicroscopy" target="_blank">@openmicroscopy</a></p>
                             <p class="account-desc">Our main Twitter account for project-wide news and interest stories.</p>
                         </div>
                     </div>                        
                     <div class="social-media-feature-section-feature top-right">
-                        <a class="account-profile" href="http://www.twitter.com/bioformats" target="_blank"><img class="account-profile-pic" src="https://pbs.twimg.com/profile_images/2393978271/u1vigayc1a3ypdhpxc7u_400x400.png" alt="@bioformats twitter profile pic"></a>
+                        <a class="account-profile" href="https://www.twitter.com/bioformats" target="_blank"><img class="account-profile-pic" src="https://pbs.twimg.com/profile_images/2393978271/u1vigayc1a3ypdhpxc7u_400x400.png" alt="@bioformats twitter profile pic"></a>
                         <div>
-                            <p class="account-handle"><a href="http://www.twitter.com/bioformats" target="_blank">@bioformats</a></p>
+                            <p class="account-handle"><a href="https://www.twitter.com/bioformats" target="_blank">@bioformats</a></p>
                             <p class="account-desc">Development status feed of coming features and release announcements.</p>
                         </div>
                     </div>                        
                     <div class="social-media-feature-section-feature bottom-left">
-                        <a href="http://www.twitter.com/idrnews" target="_blank"><img class="account-profile-pic" src="https://pbs.twimg.com/profile_images/847712210067398657/IZywWoxH_400x400.jpg" alt="@idrnews twitter profile pic"></a>
+                        <a href="https://www.twitter.com/idrnews" target="_blank"><img class="account-profile-pic" src="https://pbs.twimg.com/profile_images/847712210067398657/IZywWoxH_400x400.jpg" alt="@idrnews twitter profile pic"></a>
```

Comment:
Interesting that the IDR anchors don't get the `class="account-profile"` like the other two.

Reply:
Huh, i hadn't even noticed that! It doesn't seem to make any difference to the rendered page but I could add them in for completeness. Is there a reason behind this @lunson ?

Reply:
@hflynn The right margin was off for those two so I left the class out, but if you're not seeing any difference, then yes, feel free to add the class.

Reply:
Sorry, I meant the fact that they are missing makes no difference to how they look compared with the others. If it made the margin off, we can certainly leave it as is.